File: bwl.Imaging/Overlay/BitmapObject.vb
Public Class BitmapObject

    Public Property RectangleF As RectangleF

    Public Property Bitmap As Bitmap

    Public Sub New()

    End Sub

    Public Sub New(bitmap As Bitmap, rectangle As RectangleF)
        Me.RectangleF = rectangle
        Me.Bitmap = bitmap
    End Sub
    Public Sub New(bitmap As Bitmap, x1 As Single, y1 As Single, x2 As Single, y2 As Single)
        Me.RectangleF = RectangleF.FromLTRB(x1, y1, x2, y2).ToPositiveSized
        Me.Bitmap = bitmap
    End Sub
End Class

File: Bwl.Imaging/Bitmap/BitmapObject.cs
using System.Drawing;

namespace Bwl.Imaging
{

    public class BitmapObject
    {
        public RectangleF RectangleF { get; set; }
        public Bitmap Bitmap { get; set; }

        public BitmapObject()
        {
        }

        public BitmapObject(Bitmap bitmap, RectangleF rectangle)
        {
            RectangleF = rectangle;
            Bitmap = bitmap;
        }
        public BitmapObject(Bitmap bitmap, float x1, float y1, float x2, float y2)
        {
            RectangleF = RectangleF.FromLTRB(x1, y1, x2, y2).ToPositiveSized();
            Bitmap = bitmap;
        }
    }
}